### PR TITLE
add linkedin and custom website fields

### PR DESCRIPTION
**Pull request by jlstrater**
resolves issue #11 

Also, fixes the problem with links in the Twitter and GitHub fields.

### CODE REVIEW

File: web/public/index.html [hunk 1]
```diff
@@ -58,11 +75,17 @@
         <input type="hidden" name="redirect_uri" value="https://groovycommunity.com/success_signup.html">
         <input type="hidden" name="team_id" value="T2LNDB8SG">
 
+        <p>We verify all invitation requests. This information is not retained after the verification process. If we
```
Comment (danveloper) [inline web/public/index.html:78]: SHOULDN'T THIS BE INSIDE OF CLIPPY?!

Comment (staticnull) [reply web/public/index.html:78]: Don't worry I'll be making a PR to remove clippy.

File: web/public/index.html [hunk 2]
```diff
@@ -58,11 +75,17 @@
...
+          are unable to find your email address associated with one of the links provided, an employer's custom domain,
+          or the groovy mailing list your request may be delayed or denied. Also, users are limited to one account per
```
Comment (bobbywarner) [inline web/public/index.html]: Please capitalize Groovy.

File: web/public/index.html (hunk 1 above)
Comment (jlstrater) [reply web/public/index.html:78]: Also there is too much text to put inside of clippy.

**Pull request merged**